pyrk/.coverage

```diff
@@ -0,0 +1 @@
ies/exceptions.py":[],"/Users/gbremer/projects/pyrk/pyrk/reactivity_insertion.py":[],"/Users/gbremer/projects/pyrk/pyrk/materials/trisos.py":[],"/Users/gbremer/projects/pyrk/pyrk/materials/ss316.py":[],"/Users/gbremer/projects/pyrk/pyrk/utilities/ur.py":[],"/Users/gbremer/projects/pyrk/pyrk/db/database.py":[],"/Users/gbremer/projects/pyrk/pyrk/th_component.py":[],"/Users/gbremer/projects/pyrk/pyrk/materials/material.py":[],"/Users/gbremer/projects/pyrk/pyrk/timer.py":[],"/Users/gbremer/projects/pyrk/pyrk/materials/flibe.py":[],"/Users/gbremer/projects/pyrk/pyrk/ver.py":[34,35,37,40,41,42,43,44,45,46,52,53,54,56,57,58,61,67,73,75,78,81,85,89,94,98,101,102,103,106,107,108,109,112],"/Users/gbremer/projects/pyrk/pyrk/inp/tests/test_sim_info.py":[1,3,5,6,8,9,10,13,15,16,19,21,24,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,57,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,85,86,87,88,89,90,91],"/Users/gbremer/projects/pyrk/pyrk/data/tests/test_precursors.py":[1,4,5],"/Users/gbremer/projects/pyrk/pyrk/materials/sfrmetal.py":[],"/Users/gbremer/projects/pyrk/pyrk/neutronics.py":[],"/Users/gbremer/projects/pyrk/pyrk/data/tests/test_decay_heat.py":[1,3,6,7,8,9,10,11,12,13,14,15,16,19,20,21,22,23,24,25,26,29,31,32,33,34,35,36,37,40,42,43,44,45,46,47,48],"/Users/gbremer/projects/pyrk/pyrk/__init__.py":[8,9,5,6],"/Users/gbremer/projects/pyrk/pyrk/utilities/logger.py":[],"/Users/gbremer/projects/pyrk/pyrk/materials/liquid_material.py":[],"/Users/gbremer/projects/pyrk/pyrk/materials/__init__.py":[],"/Users/gbremer/projects/pyrk/pyrk/materials/sodium.py":[],"/Users/gbremer/projects/pyrk/pyrk/inp/validation.py":[],"/Users/gbremer/projects/pyrk/pyrk/inp/__init__.py":[4],"/Users/gbremer/projects/pyrk/pyrk/inp/tests/test_validation.py":[1,3,4,7,8,9,10,11,14,15,16,17,18,21,22,23,24,25,28,29,30,31,32,35,36,37,38,39,42,43,44,45,46,49,50,51,52,53,56,57,58,59,60,63,64,65,66,67,70,71,72,73],"/Users/gbremer/projects/pyrk/pyrk/density_model.py":[],"/Users/gbremer/projects/pyrk/pyrk/materials/graphite.py":[],"/Users/gbremer/projects/pyrk/pyrk/materials/fhrfuel.py":[],"/Users/gbremer/projects/pyrk/pyrk/data/tests/__init__.py":[4],"/Users/gbremer/projects/pyrk/pyrk/data/__init__.py":[4],"/Users/gbremer/projects/pyrk/pyrk/inp/sim_info.py":[2,4,5,6,7,8,11,12,15,16,17,18,19,20,21,22,23,24,25,26,27,28,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,73,75,76,78,79,81,84,85,86,87,88,89,90,92,93,94,95,96,97,98,101,107,108,109,111,114,115,116,117,118,119,120,122,128,131,132,134,152,156,157,158,160,162,163,165,166,167,169,170,171,172,173,174,176,180,181,182,184,187,188,189,190,191,192,193,194,195,196,197,198,199,200,201,202,203,205],"/Users/gbremer/projects/pyrk/pyrk/data/precursors.py":[],"/Users/gbremer/projects/pyrk/pyrk/db/__init__.py":[4],"/Users/gbremer/projects/pyrk/pyrk/data/decay_heat.py":[],"/Users/gbremer/projects/pyrk/pyrk/utilities/__init__.py":[],"/Users/gbremer/projects/pyrk/pyrk/materials/kernel.py":[]}}
```

Comment:
should this file really be added like this? I'm not used to coveralls, but surely any file committed to the repo shouldn't reference a specific user's filesystem (in this case /Users/gbremer)